src/metal.cs

```diff
@@ -5258,6 +5258,13 @@ partial interface MTLResource {
 		[Export ("isAliasable")]
 		bool IsAliasable { get; }
 
+		[Watch (10, 4), TV (17, 4), Mac (14, 4), iOS (17, 4), MacCatalyst (17, 4)]
+#if XAMCORE_5_0
+		[Abstract]
+#endif
```

Comment:
This is no longer a breaking change:

```suggestion
		[Abstract]
```